lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php

```diff
@@ -499,4 +518,24 @@ private function getCreateTableSQL(string $table) : ?string
             [$table]
         ) ?: null;
     }
+
+    /**
+     * @param string $tableName
+     *
+     * @return Table
+     */
+    public function listTableDetails($tableName)
+    {
+        $table = parent::listTableDetails($tableName);
+
+        $tableCreateSql = $this->getCreateTableSQL($tableName) ?? '';
+
+        $comment = $this->parseTableCommentFromSQL($tableName, $tableCreateSql);
+
+        if (! empty($comment)) {
```

Comment:
`empty()` cannot be used here. This way, a comment like '`0`' will be ignored.

Reply:
:+1: changed